job/server/pom.xml

```diff
@@ -124,6 +123,22 @@
         </execution>
       </executions>
     </plugin>
+
+    <!-- REST API documentation -->
+    <plugin>
+      <groupId>com.github.kongchen</groupId>
+      <artifactId>swagger-maven-plugin</artifactId>
+      <configuration>
+        <apiSources>
+          <apiSource>
+            <locations>alluxio.master</locations>
+            <templatePath>${project.parent.parent.basedir}/templates/strapdown.html.hbs</templatePath>
+            <outputPath>${project.parent.parent.basedir}/generated/job/document.html</outputPath>
```

Comment:
document.html -> index.html please for backcompat and silly web naming conventions

Reply:
Updated, thanks.